target/linux/ramips/dts/mt7628an_wavlink_wl-wn576a2.dts

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	compatible = "wavlink,wl-wn576a2", "silvercrest,SWV-733-B1", "mediatek,mt7628an-soc";
```

Comment:
> "silvercrest,SWV-733-B1"

What's that supposed to mean?

Reply:
Well, for OpenWrt this is not evaluated at all, so you could as well drop it (only the first entry is).

If you keep it, please ensure lower case for the entire compatible.